x-pack/solutions/observability/plugins/infra/public/pages/metrics/hosts/hooks/use_hosts_view.ts

```diff
@@ -107,7 +107,10 @@ const createInfraMetricsRequest = ({
   query: esQuery,
   from: dateRange.from,
   to: dateRange.to,
-  metrics: HOST_TABLE_METRICS,
+  metrics:
+    schema === 'ecs'
+      ? HOST_TABLE_METRICS
+      : HOST_TABLE_METRICS.filter((metric) => !['rxV2', 'txV2'].includes(metric)),
```

Comment:
Temporarily disabling these 2 metrics for `semconv`. Both metrics use `derivative` agg, and that agg needs to be in a `data_histogram`, which the hosts API doesn't run.